kuma/static/styles/components/payments/form.scss

```diff
@@ -10,6 +10,11 @@
         text-align: center;
     }
 
+    // Remove 'x' from email input as it overlaps the hint action
+    input[type='email']::-ms-clear {
```

Comment:
Interesting that it does it on an input of type `email`. This[1] seems to suggest that it only happens on input of type `text`. Either way, this will resolve the problem. Thanks @joshJarr 

[1] https://developer.mozilla.org/en-US/docs/Web/CSS/::-ms-clear